packages/utils/src/shorthand.ts

```diff
@@ -251,9 +251,6 @@ export const shorthandFor: Record<ShorthandProperties, true | readonly string[]>
     'border-right-width',
   ],
   'border-left': [
```

Comment:
Ran into an issue where having `borderTop` and `borderBottom` in the same component caused an ESLint violation

This was caused by `borderTop` and `borderBottom` both being listed as being shorthands for `border-block-end-*`... I don't think we have anyone mixing `borderInline*` and `borderLeft`/`borderRight`, or anyone mixing `borderBlock*` and `borderTop`/`borderBottom`, so I think these extra entries are safe to remove

Reply:
That's fair.  I'd confidently say we have no intentional usages of this `border-left` vs. `border-inline-end`.  Maybe if we explored perfect RTL support, this might need more thought, but this works for me!